libs/designer-ui/src/lib/editor/base/utils/editorToSegment.ts

```diff
@@ -37,7 +37,8 @@ const getChildrenNodesToSegments = (node: ElementNode, segments: ValueSegment[],
 export const convertStringToSegments = (
   value: string,
   nodeMap: Map<string, ValueSegment>,
-  options?: SegmentParserOptions
+  options?: SegmentParserOptions,
+  convertSpaceToNewline?: boolean
```

Comment:
Does it make sense to make this option part of `SegmentParserOptions`?

Reply:
Now that you mention it, I think it makes sense to make 'convertSpaceToNewline' a part of the SegmentParserOptions interface. I'll make the change now.